tools/apiview/emitters/typespec-apiview/package.json

```diff
@@ -53,19 +53,19 @@
     "dist/**",
     "!dist/test/**"
   ],
-  "dependencies": {
-    "@azure-tools/typespec-azure-core": "0.29.0",
-    "@azure-tools/typespec-autorest": "0.29.0",
-    "@typespec/compiler": "0.43.0",
-    "@typespec/http": "0.43.1",
-    "@typespec/rest": "0.43.0",
-    "@typespec/versioning": "0.43.0"
+  "peerDependencies": {
+    "@azure-tools/typespec-azure-core": ">=0.26 <1.0",
+    "@azure-tools/typespec-autorest": ">=0.26 <1.0",
+    "@typespec/compiler": ">=0.40 <1.0",
+    "@typespec/http": ">=0.40 <1.0",
+    "@typespec/rest": ">=0.40 <1.0",
+    "@typespec/versioning": ">=0.40 <1.0"
   },
   "devDependencies": {
-    "@typespec/eslint-plugin": "~0.43.0",
-    "@typespec/library-linter": "~0.43.0",
-    "@typespec/prettier-plugin-typespec": "^0.43.0",
-    "@typespec/eslint-config-typespec": "~0.6.0",
+    "@typespec/eslint-plugin": ">=0.40 <1.0",
```

Comment:
devDependencies can stay pinned

Reply:
I feel like it makes sense to let these typespec ones float even though they are dev dependencies. Presumably if the eslint config or prettier plugin get updated we'd want to move to them, and we often don't know they've been released since they are announced like the core packages are.